### ISSUE TITLE
docker swarm doesn't work, it seems the server got some trobble

### ISSUE THREAD

**Issue by zaxfenfeiyu** [docker/swarm#2678]
I use docker swarm in my virtual machine many days, and it works very well ,
but suddenly, when I create the manager and the node repeatly, it seems get some trobble.

when I use  docker -H tcp://192.168.182.10:2376 info 
I can't find any nodes , and I see the container's log ,it says:

`time="2017-04-23T00:37:29Z" level=error msg="Discovery error: Failed to fetch entries, Discovery service returned 503 HTTP status code" `

I want try a new one ,but when I use docker run --rm swarm create 
I also can't get token 

```
ariel@node1:~$ docker run --rm swarm create
<html><body><h1>503 Service Unavailable</h1>
No server is available to handle this request.
</body></html>
```
what happened with the server, it's my problem or the server's?
and the container isn't independent ？ why it needs to request the server?

**Comment by cmendesce** [comment]
I get the same issue when I try to create a cluster.

When I run `$ docker run swarm create`, I get the following message:
`503 Service Unavailable. No server is available to handle this request.`

**Comment by cmendesce** [comment]
This [line](https://github.com/docker/swarm/blob/master/discovery/token/token.go#L134) is trying to request a post in the following url `https://discovery.hub.docker.com/v1`, but it's unavailable.

**Comment by cmendesce** [comment]
There is a related issue here [2678](https://github.com/docker/swarm/issues/2675)

**Comment by zaxfenfeiyu** [comment]
I use another way to create cluster.And it works well.
I didn't use token again. This time ,I use progrium/consul.

**Comment by cmendesce** [comment]
This pull request was merged #2679 and I think this issue can be closed.
